frontend/app/forms/application-for-board-of-pardon-e/application-for-board-of-pardon-e.pdf.component.js

```diff
@@ -0,0 +1,230 @@
+          <CurrentDate left="68.12%" top="42.8%" />
+          <PositionedCheckmark
+            debugKey="check"
+            left="5.90%"
+            top="85.7%"
+            shouldShow={data.paymentMethod === "check"}
+          />
+          <PositionedCheckmark
+            debugKey="creditCard"
+            left="5.90%"
+            top={creditCardTop}
+            shouldShow={data.paymentMethod === "creditCard"}
+          />
+          <PositionedCheckmark
+            debugKey="visa"
+            left="15.82%"
+            top={creditCardTop}
+            shouldShow={
+              data.paymentMethod === "creditCard" &&
+              data.creditCardIssuer === "visa"
+            }
+          />
+          <PositionedCheckmark
+            debugKey="mastercard"
+            left="22.72%"
+            top={creditCardTop}
+            shouldShow={
+              data.paymentMethod === "creditCard" &&
+              data.creditCardIssuer === "mastercard"
+            }
+          />
+          <PositionedCheckmark
+            debugKey="discover"
+            left="33.72%"
+            top={creditCardTop}
+            shouldShow={
+              data.paymentMethod === "creditCard" &&
+              data.creditCardIssuer === "discover"
+            }
+          />
+          <PositionedCheckmark
+            debugKey="amex"
+            left="43.1%"
+            top={creditCardTop}
+            shouldShow={
+              data.paymentMethod === "creditCard" &&
+              data.creditCardIssuer === "amex"
+            }
+          />
+          {data.paymentMethod === "creditCard" && creditCardInfo({ data })}
+          {data.paymentMethod === "creditCard" && (
+            <PositionedString
+              dataKey="nameOnCard"
+              left="33.61%"
+              top={nameOnCardTop}
+            />
+          )}
+          {data.paymentMethod === "creditCard" && (
+            <React.Fragment>
+              <PositionedString
+                dataKey="cardZip"
+                left="79.12%"
+                top={nameOnCardTop}
+              />
+              <CurrentDate left="78.12%" top="84.12%" />
+            </React.Fragment>
+          )}
+        </Scoped>
+      </RenderPage>
+    </React.Fragment>
+  );
+}
+
+function CurrentDate(props) {
+  return (
+    <PositionedString debugKey="todaysDate" {...props}>
+      {moment().format("L")}
+    </PositionedString>
+  );
+}
+
+function creditCardInfo({ data }) {
+  const cardNumber = data.cardNumber || "";
+  const individualCardNumbers = cardNumber.split("").slice(0, 16);
+
+  const securityNumber = data.cardSecurityNumber || "";
+  const individualSecurityNumbers = securityNumber.split("").slice(0, 4);
+
+  const expMonthDay =
+    (data.cardExpirationMonth || "") + (data.cardExpirationYear || "");
+  const individualExpNumbers = expMonthDay.split("").slice(0, 4);
+
+  return (
+    <Scoped css={css}>
+      {individualCardNumbers.map((number, index) => {
```

Comment:
wow implementing out the credit card section was actually not necessary, but I'm impressed you did and we'll definitely keep it / merge it.